vendor/licenses/grammar/ABNF.tmbundle.txt

```diff
@@ -1,9 +1,10 @@
 ---
 type: grammar
 name: ABNF.tmbundle
+version: 0042b3d7ee21b2f800a8f80d42761513f16d9a52
 license: apache-2.0
 ---
-                                 Apache License
+Apache License
```

Comment:
🤦‍♂️ ok `licensed` is too aggressive in stripping leading empty space

Reply:
Oops. I meant to update our Licensed dependency too. If you're going to address this 🔜, I can hold off merging this PR and update the licenses again once a newer version of Licensed is available.

Reply:
Hmm, I think you may not need to do anything @jonabc. Just updated the gemspec and I'm not seeing that over-stripping of whitespace with Licensed 1.3.3.

Reply:
Hmmm, but now the version has changed 😕 Issue in Licensed coming up.